force_wfmanager/central_pane/plot.py

```diff
@@ -126,19 +99,17 @@ class Plot(HasStrictTraits):
         )
     )
 
+    update_required = Bool(False)
+
+    def _plot_updater_default(self):
+        return Timer(1000,
```

Comment:
This is the way around #270 , currently hardcoded to update the plot once a second.